src/main/java/net/unicon/lti/controller/lti/OIDCController.java

```diff
@@ -74,6 +77,8 @@ public String loginInitiations(HttpServletRequest req, Model model) {
 
         // We need to receive the parameters and search for the deployment of the tool that matches with what we receive.
         LoginInitiationDTO loginInitiationDTO = new LoginInitiationDTO(req);
+        //Enable this line to test only cookies
+        //loginInitiationDTO.setLtiStorageTarget(null);
```

Comment:
Could you make this configurable via the application.properties file?

Reply:
Good idea